libs/libcommon/src/libcommon/orchestrator.py

```diff
@@ -787,6 +790,21 @@ def finish_job(
         processing_step = processing_graph.get_processing_step_by_job_type(job_info["type"])
     except ProcessingStepDoesNotExist as e:
         raise ValueError(f"Processing step for job type {job_info['type']} does not exist") from e
+
+    try:
+        previous_response = get_response_metadata(
+            kind=processing_step.cache_kind, dataset=params["dataset"], config=params["config"], split=params["split"]
+        )
+        previous_failed_runs = 0 if previous_response is None else previous_response["failed_runs"]
```

Comment:
hmmm: it cannot be None, if I understand correctly

Reply:
Correct, I fixed it. Thanks!